en/mulesoft/modules/ROOT/pages/ms_composer_ms_release_notes.adoc

```diff
@@ -4,6 +4,27 @@ Use MuleSoft Composer to build connected experiences faster. You don't need to w
 
 New versions are automatically available once they are released; you don't need to perform any tasks to move to the new version.
 
+== August 2, 2022
+
+include::_partials/release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+3]
+
+* In Salesforce Connector, you can now specify conditions for triggers.
+
+include::_partials/release-notes/rn-headings.adoc[tag=knownIssues,leveloffset=+3]
+
+==== Composer Platform
+
+[%header%autowidth.spread]
+
+|===
+
+|Known Issue|Workaround |ID
+
+|When deleting a connection, Composer ensures that no flows use that connection. If flows use that connection, they appear in a list in the *Delete Connection* window.  However, in rare cases, flows created or edited before July 12, 2022 may not appear in the list. | |
```

Comment:
```suggestion
|Before a connection is deleted, Composer ensures that no flows use that connection. If flows use that connection, they appear in a list in the *Delete Connection* window.  However, in rare cases, flows created or edited before July 12, 2022 may not appear in the list. | |
```

Reply:
since this is a partial I may have to do a global replace for this edit later